app/js/handlebarsScoreTable.js

```diff
@@ -19,15 +19,20 @@ async function sortUsersObjectByDate() {
  * Get the handlebars template for table rows (adminTable.hbs), combine
  * with user objects and send this to searching and filtering.
  */
-function updateScoreTable() {
-    let users = sortUsersObjectByDate();
-    users.then(function (userInfo) {
-        getTemplateAjax('js/templates/adminTable.hbs').then(function (HBTemplate) {
-            let filteredUserArray = searchAndFilter(userInfo.data);
-            updateChart(filteredUserArray);
-            printFilteredResultsToScreen(HBTemplate, filteredUserArray);
-        })
-    })
+async function updateScoreTable() {
+    let userInfo = await sortUsersObjectByDate();
+    let HBTemplate = await getTemplateAjax('js/templates/adminTable.hbs');
+    let filteredUserArray = searchAndFilter(userInfo.data);
+    let paginatedArrays = splitArray(filteredUserArray, 20);
+
+    updateChart(filteredUserArray);
+    if (paginatedArrays.length > 1 ){
+        printFilteredResultsToScreen(HBTemplate, paginatedArrays[0]);
+    } else {
+        printFilteredResultsToScreen(HBTemplate, paginatedArrays);
```

Comment:
surely because of how `splitArray` works, `paginatedArrays` will always be an array of arrays, so you always want the first one?

Reply:
this is so that on page load it always displays the first page unless there is no data.

Reply:
it would error undefined when we tried to pass paginatedArrays[o] if it was empty